src/coordinator/users.spec.js

```diff
@@ -93,7 +97,7 @@ describe('Coordinator/Users', function() {
       });
 
       it('posts the new user to the server', function() {
-        expect(request.post).to.be.calledWith(
+        expect(axiosStub).to.be.calledWith(
```

Comment:
Just FYI, I think you could drop the `axiosStub` variable and just have

`sinon.stub(axios, 'post').resolves();`

on its own and on this line you could have

`expect(axios.post).to.be.calledWith(`

and it would work just the same

Reply:
Awesome tip, thanks!